client/js/clientLogic.js

```diff
@@ -0,0 +1,280 @@
+    restaurantsAPI.getOrganizationReservations(organization,
+      function(reservationsResponse) {
+        drawModule.createReservationsTable(reservationsResponse);
+      },
+      function(error) {
+        alert('Cannot get user reservations for: ' + organization);
+        console.log(error);
+      });
+  }
+
+  function createNewReview(name, rating, description) {
+
+    restaurantsAPI.createNewReview(name, rating, description,
+      drawModule.closePopUpWindow,
+      function(err) {
+        alert('Cannot add review');
+        console.log(err);
+      }
+    );
+  }
+
+  function updateReview(reviewId, rating, description) {
+
+    restaurantsAPI.updateReview(reviewId, rating, description,
+      drawModule.closePopUpWindow,
+      function(err) {
+        alert('Cannot update review');
+        console.log(err);
+      }
+    );
+  }
+
+  function showReview(reviewId) {
+
+    restaurantsAPI.getReview(reviewId, function(reviewResponse) {
+      //'Edit review ' + ' for ' + review.itemReviewed.name;
+      var restaurantReviewed =
+        JSON.parse(reviewResponse)[0].itemReviewed.name;
+      var reviewDiv = drawModule.createViewReviewDiv(reviewResponse);
+      drawModule.setPopupTitle('Review for ' + restaurantReviewed);
+      drawModule.setPopupContent(reviewDiv);
+      drawModule.openPopUpWindow();
+    },
+    function(err) {
+      alert('Cannot get the specified review');
+      console.log(err);
+    });
+
+  }
+
+  function showUpdateReviewForm(reviewId) {
+    restaurantsAPI.getReview(reviewId,
+      function(reviewResponse) {
+        var review = JSON.parse(reviewResponse)[0];
+        var formDiv =
+          drawModule.createReviewForm(review.itemReviewed.name, review);
+        drawModule.setPopupTitle(
+          'Edit Review for ' + review.itemReviewed.name);
+        drawModule.setPopupContent(formDiv);
+        drawModule.inicializeReviewForm(review);
+        drawModule.openPopUpWindow();
+      },
+      function(err) {
+        alert('Error retrieving the review');
+        console.log(err);
+      });
+  }
+
+  function createNewReservation(name, partySize, time) {
+
+    restaurantsAPI.createNewReservation(name, partySize, time,
+      drawModule.closePopUpWindow,
+      function(err) {
+        alert('Cannot add reservation');
+        console.log(err);
+      }
+    );
+  }
+
+  function deleteReview(reviewId) {
+    restaurantsAPI.deleteReview(reviewId,
+      function() {
+        location.reload();
+      },
+      function(err) {
+        alert('Could not delete the reservation.');
+        console.log(err);
+      });
+  }
+
+  function cancelReservation(reservationId) {
+    restaurantsAPI.cancelReservation(reservationId,
+      function() {
+        location.reload();
+      },
+      function(err) {
+        alert('Could not delete the review.');
+        console.log(err);
+      });
+  }
+
+
+  /* This function sets the method to be used for the differents
```

Comment:
"[...] to be used for the differents [...]" -> "[...] to be used for the different [...]"